### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by EgoziE**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,13 +4,20 @@ Thanks for contributing. We're happy for your input and would ask you to follow
...
+```bash
+git clone git@github.com:wix/wix-style-react.git
+cd wix-style-react
+npm install
+```
+
 Let's get the storybook running locally
 ```bash
-npm start
+npm run build
```
Comment (argshook) [inline CONTRIBUTING.md:16]:
```suggestion
npm start
```
it's enough, if build has never been run script will run it automatically

Comment (EgoziE) [reply CONTRIBUTING.md:16]: Cool, I didn't know that myself.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,13 +4,20 @@ Thanks for contributing. We're happy for your input and would ask you to follow
...
+```bash
+git clone git@github.com:wix/wix-style-react.git
+cd wix-style-react
+npm install
+```
+
 Let's get the storybook running locally
 ```bash
 npm start
+npm run storybook
```
Comment (shlomitc) [inline CONTRIBUTING.md:17]: `npm start` is enough, no need to run storybook.

File: docs/contribution/ADDING_ICONS.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Adding icons
 
-1. Receive the approval for the new icon from one of the library's UXs [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)
+1. Receive the approval for the new icon from one of the library's UXs [milkyfruit](https://github.com/milkyfruit) or [ZilvinasMaciulis](https://github.com/ZilvinasMaciulis)
 1. Go to [wix-ui-icons-common](https://github.com/wix/wix-ui/tree/master/packages/wix-ui-icons-common) package and follow the guidelines.
 1. After a new `wix-ui-icons-common` version is published, update `wix-ui-icons-common`'s version to the new one in `package.json` (if you are using new icon in `wix-style-react` project).
```
Comment (shlomitc) [inline docs/contribution/ADDING_ICONS.md:5]: no need to update the version, it's using `^`

File: docs/contribution/COMPONENT_STRUCTURE.md [hunk 4]
```diff
@@ -6,51 +6,50 @@ Component consists of multiple files. The following structure is the basic struc
 
 ```
 src/ComponentName
-├── index.js                                # entry file for component
-├── index.d.ts                              # type definition for the component
-├── ComponentName.js                        # component implementation
-├── ComponentName.st.css                    # component stylesheet
+└── docs
+    └── index.story.js                      # entry file for documentation
```
Comment (shlomitc) [inline docs/contribution/COMPONENT_STRUCTURE.md:10]:
```suggestion
|   └── index.story.js                      # entry file for documentation
```

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5926/

File: docs/contribution/COMPONENT_STRUCTURE.md [hunk 5]
```diff
@@ -6,51 +6,50 @@ Component consists of multiple files. The following structure is the basic struc
...
 1. Component styles and variations are tested with visual in mind. This file contains visual simulations as storybook stories.
 1. Read [Visual Testing](./VISUAL_TESTING.md.md) for more information.
 
 ### `docs/index.story.js`
-
 1. Entry file for documentation. It uses `.story` convention in order to apply automated documentation tool.
 1. Read [Component Documentation section](./DOCUMENTING_COMPONENTS.md) for more information.
+
+### `ComponentName.meta.js`
+* Am internal metadata file for automatic documentation, No action required here.
```
[the remark this answers is not in the document]
Comment (EgoziE) [reply docs/contribution/COMPONENT_STRUCTURE.md:55]: Right... We use it for visual too!

**Pull request merged**